### ISSUE TITLE
Product.Variations.UpdateWithNull not working correctly

### ISSUE THREAD

**Issue by EMC2Trooper** [XiaoFaye/WooCommerce.NET#449]
- Wordpress v5.2, WooCommerce v3.9.2 and WC.NET v0.8.1

When using the below method, an error is returned from WC with invalid and mismatched types

```
await wc.Product.Variations.UpdateWithNull(id, parentid, new
    {
        regular_price = txt1.Text,
        sale_price = txt2.Text,
        manage_stock = txt3.Text,
        sku = txt4.Text,
        stock_quantity = txt5.Text
    });
```
I have checked the inputs are the correct formats as they are populated from a .get method. The two properties throwing the error currently are manage_stock and _stock_quantity, however when commented out, the other properties like regular_price are sent as null too, therefore I believe no data is actually being sent.

See error message below:

> System.Net.WebException: '{"code":"rest_invalid_param","message":"Invalid parameter(s): manage_stock, stock_quantity","data":{"status":400,"params":{"manage_stock":"manage_stock is not of type boolean.","stock_quantity":"stock_quantity is not of type integer."}}}'

I have had success with other methods like wc.Product.UpdateWithNull and have no problem with any of the .Get(id) methods. 

Any help appreciated.

**Comment by XiaoFaye** [comment]
Maybe something like this?

```
await wc.Product.Variations.UpdateWithNull(id, parentid, new
    {
        regular_price = txt1.Text,
        sale_price = txt2.Text,
        manage_stock = int.Parse(txt3.Text),
        sku = txt4.Text,
        stock_quantity = int.Parse(txt5.Text)
    });
```

**Comment by EMC2Trooper** [comment]
Thanks for your quick reply!
Sorry that was just an example. I am actually correctly casting all the inputs before sending. Here's my actual code and the exception returned from the server.

![exception](https://user-images.githubusercontent.com/23444115/78882233-e9479400-7aab-11ea-884c-1806099ec2b0.JPG)

**Comment by XiaoFaye** [comment]
manage_stock could be int sometimes, just put raw values and see how it goes.

```
await wc.Product.Variations.UpdateWithNull(id, parentid, new
    {
        regular_price = txt1.Text,
        sale_price = txt2.Text,
        manage_stock = 1,
        sku = txt4.Text,
        stock_quantity = 0.99m
    });
```

**Comment by EMC2Trooper** [comment]
Thanks again for your help.
It seems no matter what I put in there I get the same bad response, which leads me to think it may be a bug in the package.

Have you had any success with this method?

` await wc.UpdateVariationWithNull(Int32.Parse(txtVariationID.Text),Int32.Parse(txtProductID.Text), new
    {
        regular_price = 1m,  
        sale_price = 0.5m,  
        manage_stock = 1,  
        sku = "1056BL",  
        stock_quantity = 1m  
    });`

**Comment by XiaoFaye** [comment]
You're right, there is a bug in the code, I will fix it in the next release.

**Comment by EMC2Trooper** [comment]
Thanks, I really appreciate it!

**Comment by EMC2Trooper** [comment]
Hi James, do you have any estimate when the next release might be published? I would love to buy you some more coffees if it might help :)

**Comment by XiaoFaye** [comment]
Hi, this has been fixed in the latest version 0.8.2, you should be able to download from nuget now.

**Comment by EMC2Trooper** [comment]
You're awesome! thanks again mate.
